openapi.yaml

```diff
@@ -156,6 +156,7 @@ paths:
          @@team_name1@@ | Resolves to the recipient’s ‘Team Name’ field.<br/><br/> For example, Red Team. 
          @@messagelabel@@| Resolves to the label field of the sent message. <br/><br/>For example, Incident #1234. 
          @@messagecategories@@ | Resolves to the list of categories used in the message, separated with commas. <br/><br/>For example, Product Update. 
+         @@web_link@@ | Resolves to a generated unsuubscribe link. <br /><br />For example, <pre>https://<region>.whispir.com/unsubscribe?_parameters=value</pre>
```

Comment:
typo in unsubscribe word

Reply:
also, @@web_link@@ is not always for unsubscribe - it is generally used as a link to the web channel content.  We just reused it for unsubscribe

Reply:
I'll resolve this comment :) @lnachor-whispir